bin/testclient.py

```diff
@@ -67,6 +68,9 @@ def main():
         # plant.day_overview(date(2016, 2, 3), quarter=False)
         # plant.month_overview(date(2016, 1, 1))
         # plant.year_overview(date(2016, 2, 1))
+        # for entry in plant.logbook(date(2016, 2, 1)).entries:
+        #    description = unescape(entry['description'])
```

Comment:
Have you seen logs where this was needed? What did they look like? If this is really needed, then perhaps it should be made when decoding the response instead.

Reply:
Yes. Here is a sample:

> 
Yield deviation from inverter comparison &apos;&apos;SB 4500TL-JP-22 959&apos;&apos;, serial number: &apos;&apos;1310395981&apos;&apos;, total yield 7.842 kWh, spec. yield 1.65 kWh/kWp deviation 23.8579% (&gt;20%) from the average of the monitored inverters (2.167 kWh/kWp on 11/23/2020).

I hesitated to put it in the response decoding step because I wasn't sure if importing an additional module was OK.
I can make the change easily enough if you'd like.

Reply:
Perhaps you can use `xml.sax.saxutils.unescape` (with apos and quot parameter as [described here](https://wiki.python.org/moin/EscapingXml)). That would make slightly more sense, since the rest of the response is XML.

Reply:
Sorry for the delay.
I'll give it a try right now.

Reply:
Changes made.
I looked through two days' worth of logs for 12 different PV plants and everything seems to work well. There are no other entities that need escaping that I'm aware of.